CMakeLists.txt

```diff
@@ -41,23 +41,17 @@ macro(build_uncrustify)
   # Pinning to tip of master at the time of a desired bugfix
   set(uncrustify_version 45b836cff040594994d364ad6fd3f04adc890a26)
   # Get uncrustify 0.68.1
-  ExternalProject_Add(uncrustify-0.68.1
+  ExternalProject_Add(uncrustify-${uncrustify_version}
     GIT_REPOSITORY https://github.com/uncrustify/uncrustify.git
     GIT_TAG ${uncrustify_version}
-    GIT_CONFIG advice.detachedHead=false
```

Comment:
Can we keep this line?  It doesn't get rid of all of the stderr output (there is apparently no way to do that in `git` without access to the command-line), but it does suppress the worst of it.

Reply:
Sure, I just figured it was part of the patching change and could be dropped.

Reply:
Fixed in [force-push](https://github.com/ament/uncrustify_vendor/compare/dc42d53553cc456d03ffff082f71e550593b1041..7a90b62c40b6ddfc53c35dbff21f3050e4d3ca4b).